db/migrate/20170525091520_set_restaurant_owner_attribute_to_default_false.rb

```diff
@@ -0,0 +1,5 @@
+class SetRestaurantOwnerAttributeToDefaultFalse < ActiveRecord::Migration[5.0]
+  def change
+    change_column :users, :restaurant_owner, :boolean, :default => false
+  end
+end
```

Comment:
This could have been (and should be) achieved in ONE migration. Also, please use the new hash syntax. (No hash-rockets please)